tock/tock/templates/utilization/group_utilization.html

```diff
@@ -4,132 +4,129 @@
+                {{ userdata.last }}
+                ({{ userdata.last_billable_hours_total }} /
+                {{ userdata.last_all_hours_total }})
+              </a>
+            {% else %}
+                --
+            {% endif %}
+            </td>
+            <td>
+            {% if userdata.recent_all_hours_total %}
+              {{ userdata.recent }}
+              ({{ userdata.recent_billable_hours_total }} /
+              {{ userdata.recent_all_hours_total }})
+            {% else %}
+              --
+            {% endif %}
+            </td>
+            <td>
+            {% if userdata.fytd_all_hours_total %}
+              {{ userdata.fytd }}
+              ({{ userdata.fytd_billable_hours_total }} /
+              {{ userdata.fytd_all_hours_total }})
+            {% else %}
+              --
+            {% endif %}
+            </td>
+          </tr>
         {% endfor %}
-        </b>
-      </td>
-      <td>
-        <b>
-        {% for ut in unit_totals %}
-        {% if i.1 is ut.last.unit_name %}
-          {{ ut.recent.utilization }}<br>
-          ({{ ut.recent.billable_hours }} / {{ ut.recent.total_hours }})
-        {% endif %}
-        {% endfor %}
-        </b>
-      </td>
-      <td>
-        <b>
-        {% for ut in unit_totals %}
-        {% if i.1 is ut.last.unit_name %}
-          {{ ut.fytd.utilization }}<br>
-          ({{ ut.fytd.billable_hours }} / {{ ut.fytd.total_hours }})
-        {% endif %}
-        {% endfor %}
-        </b>
-      </td>
-    </tr>
-  </tfoot>
-  <tbody>
-  {% for userdata in object_list %}
-
-  {% if userdata.unit is i.0 %}
-    <tr class="report_table__row">
-      <th scope="row">
-        {{ userdata.user_data }}
-      </th>
-      <td>
-      {% if userdata.last_all_hours_total %}
-        <a href='{{ userdata.last_url }}' title="Percent billable (billable hours / total hours)">
-          {{ userdata.last }}
-          ({{ userdata.last_billable_hours_total }} /
-          {{ userdata.last_all_hours_total }})
-        </a>
-      {% else %}
-          --
-      {% endif %}
-      </td>
-      <td>
-      {% if userdata.recent_all_hours_total %}
-        {{ userdata.recent }}
-        ({{ userdata.recent_billable_hours_total }} /
-        {{ userdata.recent_all_hours_total }})
-      {% else %}
-        --
-      {% endif %}
-      </td>
-      <td>
-      {% if userdata.fytd_all_hours_total %}
-        {{ userdata.fytd }}
-        ({{ userdata.fytd_billable_hours_total }} /
-        {{ userdata.fytd_all_hours_total }})
-      {% else %}
-        --
-      {% endif %}
-      </td>
-    </tr>
-  {% endif %}
-  {% endfor %}
-  </tbody>
-</table>
-
-{% endfor %}
+        </tbody>
+      </table>
+    {% endfor %}
 
-{% else %}
+  {% else %}
 
-<h3>This page is viewable by Tock users with an `is_staff` status of True.</h3>
-<p>If you need access, please contact your supervisor.</p>
+    <h3>This page is viewable by Tock users with an `is_staff` status of True.</h3>
```

Comment:
I know this was already here previously. But I don't see the value in including the `is_staff` property for users. I think this copy could be cleaned up to read something like.

### You don't have permission to view this information.
If you need access, please contact your supervisor.

Reply:
I agree, Roger. I actually pondered moving the is_staff check up into the view so we could and redirecting them before making all the queries, but in the end I didn't expand the scope to do that. I'm not averse to doing so, though, though.

I think the main  thing is to decide if you want this page handling both, or if we'd rather redirect to some sort of 403 template. My vote would be the latter, but either will work.